Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -87,6 +99,32 @@
                 />
             </b-row>
         </div>
+        <div
+            v-if="filteredTimelineEntries.length == 0"
+            class="text-center pt-2"
+        >
+            <b-row>
+                <b-col>
+                    <img
+                        class="mx-auto d-block"
+                        src="@/assets/images/timeline/empty-state.png"
+                        width="200"
+                        height="auto"
+                        alt="..."
+                    />
+                </b-col>
+            </b-row>
+            <b-row>
+                <b-col>
+                    <p
+                        id="noTimelineEntriesText"
```

Comment:
does it need to be an id? a class should be better since is just being used for the style of p

Reply:
changed it to a class